platform/administer/monitoring/central-hostpath-mapper.mdx

```diff
@@ -93,8 +92,7 @@ below and configuration should be done via the cli or by changes to manifests.
         Toggle the slider <Label>Enable</Label>. This will add the
         'loft.sh/hpm-enabled' annotation to the template metadata which will be
         automatically synced to the virtual cluster on creation. Secondly, it
-        will add the 'rewrite-host-paths=true' as extraArgs to the helm values
-        for the syncer.
+        will add 'enabled: true' & 'central: true' under the controlPlane.hostPathMapper field in the vcluster.yaml.
```

Comment:
It could be helpful to add make `controlPlane.hostPathMapper` a link to the `.../control-plane/components/host-path-mapper.mdx` so they can jump straight to that if they don't know about that part of `vcluster.yaml`

Reply:
done